app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,6 @@
+class ReviewsController < ApplicationController
+
+  def index
+    @reviews = Review.order("id DESC").all  # ordered by most recent
```

Comment:
.all here isn't necessary, .order already returns all unless you further narrow it otherwise.